src/Symfony/Component/Notifier/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * Add PHPUnit constraints
  * Add `from` property in `SmsMessage`
+ * Change incorrect message, when the sender in the global envelope or the from header of asEmailMessage() is not defined.
```

Comment:
Should be removed, we do not add such changes in the CHANGELOG, which is only for new features and deprecations.

Reply:
Ok :)